hydroshare/settings.py

```diff
@@ -566,4 +572,107 @@
+        "'sha256-JCBsts/37Jx84rU5noLWawBDCAgz9kEjdmJQN3jBY8k='",
+        "'sha256-04T2hHmvLBivvYNrvZCsJi3URODWHuMDbrtYi3CIfB4='",
+        "'sha256-DC3munJ0pghuoA4hX8dh32935FOMe4Ek0lEToguPh04='",
+        "'sha256-NDFP8wAl44R2n9vT1corxoEbvzy3cusyeGupfuQ1U0c='",
+        "'sha256-PA1G79Xx6LLYGxSPHSieelZ8bBLAWIMgD/XXawZp7qU='",
+        "'sha256-mCQxt+MP+ovw3u4TQgt01msH6eqt4mSVB0Qu2YWWMTg='",
+        "'sha256-JvKBe3eX3y4VMRRoZ6gtD3NERM2ie6izqcfCYApav2Y='",
+        "'sha256-TxuSliz/loxO7gZryrQvb0iCfYpUhvSaFj/6Td2gWFQ='",
+        "'sha256-45pN/AJ0kQzJz9vwICzvV3MnuOG8gtGxggQwABWnymA='",
+        "'sha256-obaP6XgYiPObVeZTvXdVlgt809T2Dm6PIk16JE30820='",
+        "'sha256-X48Mhnt413YPDQKQ1WJRKySArZrDCn1RHfPwUz3f1n8='",
+        "'sha256-4xP5qvZtoBaEfC9gZ43CQp8bEku+/CZOsq9FaRvF7OU='",
+        "'sha256-mZcvwmv6hKtoYTrQMxgb/EQ337dAnjIahuJ3pleT24M='",
+        "'sha256-DkOZu61D7CazmSXGaEnSn2z9djJ8MykUT8DxDGQdjbs='",
+        "'sha256-DkOZu61D7CazmSXGaEnSn2z9djJ8MykUT8DxDGQdjbs='",
+        "'sha256-rkLuhOrYkT+nja4WGqG4TPLui5JaWiPDYJJ8UCuslJw='",
+        "'sha256-LvGubOm7HawnJzw+vOyACi6DYfal1wKpJYCE7KN/XDI='",
+        "'sha256-5tQ8oYGMvQerAFL7X6FcoOun/fzsYLKwBeMevN0pth8='",
+        "'sha256-wwXL5J3dFy1OlT0B3+GEak+gfFt97tLZtgA03Ww5uKU='",
+        "'sha256-jZ0Oc9ZvtRDz3fu+52erC3krRmxFNM123/clxHqXT6I='",
+        "'sha256-5SxuHuYxmlu7rNBznCkw0RTE+ONtAqVChmyV8gMsnyM='",
+        "'sha256-fH3rM69L3MlQuLHwmENXZ9SSP9u+7mECRGOpE5pY/Hw='",
+        "'sha256-7Xlpbhi2uHJajOb377ImeVoPP2wXatX9zj+Yr/DS0qA='",
+        "'sha256-4rt4xlh501T8mF5LZkGs/NIyq3fs7igdd8csZexAN8I='",
+        "'sha256-daEdpEyAJIa8b2VkCqSKcw8PaExcB6Qro80XNes/sHA='",
+        "'sha256-j91bFxk70pyBgSjXdJizPOSoYZaC0Zt+fq9U4OF/npQ='",
+        "'sha256-ZWirjyeue1OSyGpYxoRjbS2NRwKf/b/XZiYaHuUV6Wc='",
+        "'sha256-rQJ6epAwyFQxC0sb4uq4sgIKJLr2jP19K4M0Bork7kM='",
+        "'sha256-hH6THo5mChVoQ5wrDhew4wuTZxOayUnEJC3U1SK80VQ='",
+        "'sha256-fUjKhLcjxDsHY0YkuZGJ9RcBu+3nIlSqpSUI9biHAJw='",
+        "'sha256-7ydyyMhpPIo0fTHZtxmllQ+MJpMVM299EkUKAf0K1hs='"
+    ],
+    "style-src" : [
+        "self",
+        "unsafe-inline",
+        "*.googleapis.com",
+        "*.bootstrapcdn.com",
+        "*.datatables.net",
+        "*.cloudflare.com "
+        # "'sha256-eg/xnzXUz6KMG1HVuW/xUt76FyF5028DbB4i0AhZTjA='",
+        # "'sha256-G/USJC1+tllSYwvERC+xNnfMa+5foeWVYBUWvwijyls='",
+        # "'sha256-Z0H+TBASBR4zypo3RZbXhkcJdwMNyyMhi4QrwsslVeg='",
+        # "'sha256-qxBJozwM44kf1mKAeiT/XkAwReBZ/To9FXKNw3bdVwk='"
+    ],
+    "img-src" : [
+        "self",
+        "*.datatables.net",
+        "*.googleapis.com",
+        "*.gstatic.com",
+        "*.google.com"
+    ],
+    "connect-src" : [
+        "self",
+        "*.github.com"
+    ],
+    "font-src" : [
+        "self",
+        "*.gstatic.com",
+        "*.bootstrapcdn.com"
+    ],
+    "frame-src" : [
+        "self",
+        "*.hydroshare.org"
+    ]
+}
+
+
+
+X_FRAME_OPTIONS = "deny"
```

Comment:
Cleanup spacing and comments here and below. Is there useful documentation that could be linked to here (like what was provided for CSP above)?